bootstrap.js

```diff
@@ -6,6 +6,9 @@
  *
  * Contributor(s):
  *   Diego Casorran <dcasorran@gmail.com> (Original Author)
+ *	 Jerone
+ *	 Zulkarnain K.
+ *	 Noitidart <noitidart@gmail.com> (Authored Contirubtion: "Install from Edit Page without Committing")
```

Comment:
[**@jerone**](https://github.com/jerone) commented on [6 jul. 2014 12:19 CEST](https://github.com/diegocr/GitHubExtIns/pull/10/files#r14574672 "2014-07-06T10:19:38Z - Replied by Github Comment Enhancer"):
> Contirubtion -> Contribution

Still saying `Contirubtion` instead of `Contribution`.

You can also remove my name from the list as I actually didn't committed any code to this addon (see https://github.com/diegocr/GitHubExtIns/pull/8, only the idea was taken, a different approach was used).